auth-api/src/auth_api/services/rest_service.py

```diff
@@ -163,11 +164,13 @@ def get(endpoint, token=None,  # pylint: disable=too-many-arguments
         return response
 
     @staticmethod
+    @cache.cached()
```

Comment:
needs to be tied to `CACHE_DEFAULT_TIMEOUT`?

Reply:
nvm:

![image](https://github.com/bcgov/sbc-auth/assets/3484109/c0113ec0-42ef-4cad-8976-e2c4a3e94521)